policies/test_test.rego

```diff
@@ -0,0 +1,21 @@
+package hacbs.contract.test
+
+test_needs_to_have_data {
+	deny == {"msg": "No test data provided"} with data.test as []
+}
+
+test_pass_with_some_data {
+	not deny with data.test as [{}]
+}
+
+test_success_data {
+	not deny with data.test as [{"result": "SUCCESS"}]
+}
+
+test_failure_data {
+	deny == {"msg": "Tests did not end with SUCCESS"} with data.test as [{"result": "FAILURE"}]
```

Comment:
Can we add a test that mixes results? e.g. `[{"result": "FAILURE"},{"result": "SUCCESS"}]`